libraries.txt

```diff
@@ -0,0 +1,16 @@
+# needed & optional software libraries
```

Comment:
This may not for the main branch or perhaps some context for the dependency list. At this stage perhaps something for notes rather than added to the code base.